jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -229,7 +233,36 @@ YUI.add('user-profile', function() {
             </span>
           </juju.components.UserProfileEntity>);
       });
-      return components;
+      return (
+        <div className="twelve-col no-margin-bottom">
```

Comment:
@anthonydillon A react component requires at least one top level element. So the div is required but I'm not sure about the classes.